src/pages/Refund.tsx

```diff
@@ -73,6 +80,7 @@ const Refund = () => {
                 })
                 .catch((e) => {
                     log.error("invalid file upload", e);
+                    setRefundInvalid(true);
                     input.setCustomValidity(t("invalid_refund_file"));
                 });
```

Comment:
Why don't we use `await` for both, the text file and the QR code scan, to avoid having the redundant logic?

Reply:
we could but i think it fine, we may do something different for both in the future,, and the logging is different btw.